openapi-v1.yaml

```diff
@@ -1494,6 +1505,28 @@ definitions:
         type: string
         example: "organization"
 
+  GroupSharing:
+    description: details for sharing a given group
+    type: object
+    properties:
+      actions:
+        description: List of permitted actions
+        type: array
+        x-omitempty: true
+        items:
+          $ref: "#/definitions/GroupActions"
+        example:
+          - read
+          - write
+      namespace:
+        description: namespace being granted group access can be a user or organization
+        type: string
+        example: "MyOrganization"
+      namespace_type:
```

Comment:
Is this useful for some reason? Since namespaces are unique anyway.

Reply:
This is the same as ArraySharing. It is used with `getGroupSharingPolicies` to return to user the namespace type.